content/doc/addons/mongodb/_index.md

```diff
@@ -78,4 +78,20 @@ Note that these features are available for all our databases add-ons, in additio
 
 **The managed databases provide an advantage if you want to avoid the complexity of installing and deploying databases** and if you want simple-to-use management and monitoring tools.
 
-{{% content/managed-services %}}
+## 🔑 Rights and permissions
+
+Add-ons are managed services, meaning that users have **standard access** to the database (**owner** of the database). Some operations like database and user creation, as well as some settings modifications aren't available by default. This ensures optimal performances and security for managed services as configured by Clever Cloud.
+
+If you think your system might require more advanced administrative access, contact Clever Cloud support to explain your use case and we will work with you to find a solution.
+
+Here is the list of actions that you won't be able to perform:
+- Database administration (for example you won't be able to create new databases).
+- Users administration (you won't be able to create other users than the one handled with our control plane, ie the base owner and read-only users).
+- Server configuration update.
+- Cluster creation
+- Back-up frequency or retention control.
+
+Please refer to Clever Cloud support if you want to perform one of these actions.
+
+Authorized actions:
+- Manage collections (create, delete...)
```

Comment:
This part could be specific to each add-on. Thus, here we can use: 

```suggestion
Refer to Clever Cloud support if you want to manage collections (create, delete, etc.).
```

Maybe `refer to` could be replaced by `Ask`

Reply:
done for all four files